### ISSUE TITLE
archived field does not have default value

### ISSUE THREAD

**Issue by mikk150** [Block8/PHPCI#1116]
creating new project end up giving error, that 
```
SQLSTATE[HY000]: General error: 1364 Field 'archived' doesn't have a default value
```

**Comment by coxa** [comment]
I've got the same message today on fresh install

**Comment by tkuska** [comment]
me too

**Comment by leewillis77** [comment]
I can confirm that I get the same error when trying to create a project on a fresh install.

**Comment by inblank** [comment]
I confirm, I've got the same message today on fresh install.

**Comment by damiantomczak** [comment]
+1

**Comment by cirovargas** [comment]
+1 install today and piff...

**Comment by cirovargas** [comment]
Create a new database migration with archive nullable

**Comment by adamazing** [comment]
See @laurin1's fix (linked above): https://github.com/laurin1/PHPCI/commit/0a752f6245621ce83e66eeed4c3656054a86c31f

**Comment by Robdebert** [comment]
I got the same error today. Just installed everything a few minutes ago with composer the way it was described on the homepage.

How to fix this? I cannot add a new project :-(

**Comment by dancryer** [comment]
This is fixed in v1.7.1, which we've just released: https://github.com/Block8/PHPCI/releases/tag/1.7.1
